pkgs/jovian-greeter/greeter.py

```diff
@@ -145,6 +153,10 @@ def _consume_session(self) -> Optional[str]:
                 env={'SHELL': '/bin/sh'}
             )
             next_session = res.stdout.decode('utf-8').strip()
+            stderr = res.stderr.decode('utf-8').strip()
```

Comment:
Does consume-session ever log to stderr?

Reply:
Not deliberately, but the commands it run might fail, and those commands _do_ log to STDERR. In addition, it's generally a good idea to future-proof this sort of thing anyway IMO.

Reply:
Actually, we should probably use `stdout=PIPE` instead of `capture_output`. That way stderr will automatically end up in the journal.

Reply:
That's fair enough! I think that would work nicely :)